requirements/requirements.in

```diff
@@ -41,7 +43,8 @@ python-logstash==0.4.6
 python-memcached==1.58
 python-radius==1.0
 python-saml==2.2.1
-python-social-auth==0.2.21
```

Comment:
I'm so glad we had already moved to this so we didn't have to do a two-piece upgrade.